src/test/java/ReadmeExamples.java

```diff
@@ -194,7 +196,8 @@ private void batchPublisher() {
         BatchPublisher<Long, User> batchPublisher = new BatchPublisher<Long, User>() {
             @Override
             public void load(List<Long> userIds, Subscriber<User> userSubscriber) {
-                userManager.publishUsersById(userIds, userSubscriber);
+                Publisher<User> userResults = userManager.publishUsersById(userIds);
+                userResults.subscribe(userSubscriber);
             }
```

Comment:
I wanted to show how you might create a Publisher and then subscribe to it.  This is more clear in an instructional sense